### PR TITLE
Add jam-band recipe

### PR DESCRIPTION
**Pull request by eluthern**

### CODE REVIEW

File: src/recipes/jam-band.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+    "name": "<Jam Band>",
```
Comment (alfg) [inline src/recipes/jam-band.json]: You can remove all the placeholder tokens in this file:
```suggestion
    "name": "Jam Band",
```

File: src/recipes/jam-band.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    ],
+    "directions": [
+        "<Add all ingredients except club soda to shaker with ice.>",
+        "<Shake well, strain into glass filled with ice.>",
+        "<Top with club soda and garnish with mint.>"
+    ],
+    "image": "<jam-band.jpg>",
+    "source": "<https://www.cardinalspirits.com/thedrop/jam-band-cocktail-recipe>",
+    "keywords": [
+        "<keywords if available>",
```
Comment (alfg) [inline src/recipes/jam-band.json]: Add a few keywords, please. You can use the other recipe files as a reference.

File: src/recipes/jam-band.json (hunk 1 above)
Comment (eluthern) [reply src/recipes/jam-band.json]: I believe I followed requested changes, thank you for reviewing.

**Comment by alfg** [comment]
Thanks!

**Pull request merged**